setup_scripts/dagmc_setup.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+#DAGMC Installation
+INSTALL_ROOT=$HOME
+cd $INSTALL_ROOT
+mkdir dagmc
+cd dagmc
+git clone https://github.com/svalinn/DAGMC
```

Comment:
These few lines could be consolidated as `git clone -b develop https://github.com/svalinn/DAGMC` if you want